cluster/vagrant-kubernetes/setup_master.sh

```diff
@@ -46,12 +46,18 @@ done
 
 set -e
 
-if [ "$network_provider" == "weave" ]; then
+# Additional network providers are available from
+# https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/#pod-network
+case "$network_provider" in
+"weave")
     kubever=$(kubectl version | base64 | tr -d '\n')
     kubectl apply -f "https://cloud.weave.works/k8s/net?k8s-version=$kubever"
-else
-    kubectl create -f kube-$network_provider.yaml
-fi
+    ;;
+
+"flannel")
+    kubectl apply -f https://raw.githubusercontent.com/coreos/flannel/v0.9.1/Documentation/kube-flannel.yml
+    ;;
```

Comment:
should we add back the default case here?
```
*)
    kubectl create -f kube-$network_provider.yaml
    ;;
```

Reply:
I can add it, but I think the usage of some default `kube-$network_provider.yaml` is very limited because you need to download it first and remove `network_provider` variable from the config file

Reply:
I added it and so far I actually never used it. We can add it later if the need arises.